src/lib/utils/createSendEventForFacet.ts

```diff
@@ -12,19 +12,23 @@ type CustomSendEventForFacet = (customPayload: any) => void;
 export type SendEventForFacet = BuiltInSendEventForFacet &
   CustomSendEventForFacet;
 
+type CreateSendEventForFacetOptions = {
+  instantSearchInstance: InstantSearch;
+  helper: AlgoliaSearchHelper;
+  attribute: string | ((facetValue: string) => string);
+  widgetType: string;
+};
+
 export function createSendEventForFacet({
   instantSearchInstance,
   helper,
-  attribute,
+  attribute: attr,
```

Comment:
Used `attr` (instead of `_attribute` or else) so that l. 30 remains on a single line.